zimui/src/components/playlist/PlaylistGrid.vue

```diff
@@ -8,23 +9,45 @@ const { mdAndDown } = useDisplay()
 const props = defineProps<{
   playlists: PlaylistPreview[]
 }>()
+
+const items = computed(() => props.playlists.slice(0, 48))
+
+const loadMoreItems = async () => {
+  return new Promise<PlaylistPreview[]>((resolve) => {
+    setTimeout(() => {
+      resolve(props.playlists.slice(items.value.length, items.value.length + 12))
+    }, 100)
+  })
+}
+
+const load = async ({ done }: { done: (status: 'ok' | 'empty') => void }) => {
+  const res = await loadMoreItems()
+  items.value.push(...res)
+  if (items.value[items.value.length - 1] == props.playlists[props.playlists.length - 1]) {
```

Comment:
We are we comparing the last elements of items/playlists instead of just checking the length of `res`?